src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs

```diff
@@ -539,6 +539,22 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					ShowItem = UserSettingsService.GeneralSettingsService.ShowCompressionOptions && StorageArchiveService.CanDecompress(selectedItems)
 				},
 				new ContextMenuFlyoutItemViewModel()
+				{
+					Text = "Flatten".GetLocalizedResource(),
+					ShowInSearchPage = true,
+					ThemedIconModel = new ThemedIconModel()
+					{
+						ThemedIconStyle = "App.ThemedIcons.Folder",
+					},
```

Comment:
```suggestion
					ThemedIconModel = new() { ThemedIconStyle = "App.ThemedIcons.Folder" },
```

Reply:
I think this should stay the way it is currently, the coding style says to align curly brackets on new lines and the surrounding flyout items are written in the same way.

Reply:
It's ok, I was going to change here to:

new("App.ThemedIcons.Folder"),

It's easier if it is written like that I suggested.

Reply:
@devin-slothower Please change here.